packages/office-ui-fabric-react/src/components/Link/Link.types.ts

```diff
@@ -3,12 +3,19 @@ import * as React from 'react';
 /* tslint:enable:no-unused-variable */
 import { Link } from './Link';
 
+import {
+  IStyle,
+  ITheme
+} from '../../Styling';
+import { IStyleFunction } from '../../Utilities';
+
 export interface ILink {
   /** Sets focus to the link. */
   focus(): void;
 }
 
-export interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | Link> {
```

Comment:
Why was link removed here? I see the todo - was it causing an error?

Reply:
Argh, I missed my own todo comment. The TypeScript compiler is saying `'extends' clause of exported interface 'ILinkProps' has or is using private name 'Link'`. Looking into it now.

Reply:
Ah of course, this now needs to be `LinkBase` instead of `Link`. :)

Reply:
A good way to prevent this in the future is to use the rename symbol function (F2) and it smartly renames that in the whole project, then you can create the Link (or whatever component) again.